processing/dnsdumpster/dnsdumpster.py

```diff
@@ -0,0 +1,102 @@
+import os
+import sys
+import json
+
+"""
+Todo:
+    * Add json output as support_file
+    * Snag that fancy domain graph off dnsdumpster?
+    * Snag that fancy dns excel file off dnsdumpster?
+
+"""
+
+from fame.core.module import ProcessingModule
+from fame.common.constants import VENDOR_ROOT
+from fame.common.exceptions import ModuleInitializationError
+
+try:
+    sys.path.append(os.path.join(VENDOR_ROOT, 'dnsdmpstr'))
+    import dnsdmpstr
+    has_dnsdump = True
+except ImportError:
+    has_dnsdump = False
+
+try:
+    import tldextract
+    has_tldextract = True
+except ImportError:
+    has_tldextract = False
+
+class DnsDumpster(ProcessingModule):
+    name = "dnsdumpster"
+    description = "Enumerate domain DNS data."
+    acts_on = ['url']
+
+    config = [
+        {
+            'name': 'reverse_dns',
+            'type': 'bool',
+            'default': 'false',
+            'description': 'List reverse DNS entries from DnsDumpster.'
+        },
+        {
+            'name': 'page_links',
+            'type': 'bool',
+            'default': 'false',
```

Comment:
```suggestion
            'default': True,
```

This should be a boolean, and why disable this by default ?